src/components/common/menucard.tsx

```diff
@@ -12,8 +12,7 @@ type Props = {
 const MenuCard: FC<Props> = ({ menu, selectedMenu, onClick }) => {
   return (
     <Card
-      className={`menu-card grid items-center rounded-xl h-[90px] w-[158px] m-2 cursor-pointer text-center border-solid border-[#30343B] border-2 ${
-        selectedMenu === menu.id ? "selected-menu bg-[#30343B]" : "bg-[#26262B]"
+      className={`menu-card grid items-center rounded-xl h-[90px] w-[158px] m-2 cursor-pointer text-center bg-[#292E36] !shadow-none ${selectedMenu === menu.id ? "selected-menu" : ""
       }`}
```

Comment:
Can you try this 
`${selectedMenu === menu.id && "selected-menu"}`
instead of 
`${selectedMenu === menu.id ? "selected-menu" : ""}`

Reply:
Yes. It will work. its just that there will a classname "false" when the condition is false. To avoid that I have added the empty string.